spec/controllers/admin_users_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+require "rails_helper"
+
+describe Admin::UsersController do
+  context "GET #new" do
+    context "when invalid params" do
+      it "won't create a new user" do
+        user = create(:user, admin: true)
```

Comment:
Since we are creating an admin here, if we rename `user` to be `admin` it communicates better what objects/users we are working with

Reply:
Done in sha: 281932b